pyrfa/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "pyrfa" %}
+{% set version = "8.5.3" %}
+{% set fn_base = "{}-{}-cp{}-none".format(name, version, CONDA_PY) %}
+{% set fn_win = "{}-win_amd64.whl".format(fn_base) %}
+{% set fn_linux = "{}-manylinux1_x86_64.whl".format(fn_base) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ fn_win }}  # [win64]
+  fn: {{ fn_linux }}  # [linux64]
+
+  url: https://pypi.io/packages/93/bc/59b140270c7260f7bd01e20ad142351e2d9e38c03406e54582b70107089a/{{ fn_win }}  # [py==36 and win64]
```

This should be a bit easier to maintain:
```suggestion
  url: https://pypi.io/packages/{{ CONDA_PY }}/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-{{ CONDA_PY }}-none-manylinux1_x86_64.whl  # [linux64]
  sha256: 090af9e86810d01a6cd98bcdd77906085c9eeaf90cdd1618d54962dd821e65a8  # [py==27 and linux64]
  sha256: 2109d0b7f64cd76c7ad1b0e775dbfa4da7a3ac267379e4f637e0d5061dfd4f66  # [py==36 and linux64]
  sha256: 49198873d6a057c76c62f59d09104dfc93146084c41b73850211943dfd16cde2  # [py==37 and linux64]

  url: https://pypi.io/packages/{{ CONDA_PY }}/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-{{ CONDA_PY }}-none-win_amd64.whl  # [win64]
  sha256: 6cce0909df39861ef05995c6e966dd2859344d0a5ee1182fb177adf451712d6a  # [py==27 and win64]
  sha256: af3731272c75f28919f735a72c31b11f290f6355a2bd98edb56bbe6a1edce2d1  # [py==36 and win64]
  sha256: f148c1cbc2e0b47ca66b731a4154af8c901ed3209732c8704797348b70c80b88  # [py==37 and win64]
```